lib/modules/manager/flux/index.ts

```diff
@@ -1,15 +1,17 @@
+import { DockerDatasource } from '../../datasource/docker';
 import { GithubReleasesDatasource } from '../../datasource/github-releases';
 import { HelmDatasource } from '../../datasource/helm';
 import { systemManifestRegex } from './common';
 
-export { extractAllPackageFiles, extractPackageFile } from './extract';
 export { updateArtifacts } from './artifacts';
+export { extractAllPackageFiles, extractPackageFile } from './extract';
```

Comment:
why this change?

Reply:
```
npm test
...
...
/workspaces/renovate/lib/modules/manager/flux/extract.spec.ts
  2:1  error  `.` import should occur after import of `../types`  import/order

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

Reply:
VSCode on save moves it back to the position it was but than that error happens on npm test, is there a way around that?

Reply:
Fixed.